functions/packages.sh

```diff
@@ -40,6 +40,50 @@ firemotd_setup() {
   fi
 }
 
+create_mta_config() {
```

Comment:
Add function description like the one below especially explaining difference in behaviour while executed INTERACTIVE vs UNATTENDED.

https://github.com/openhab/openhabian/blob/de039dac49462990992cf127b82316202e639700/functions/passwords.sh#L3

Reply:
I believe this routine is never called unattended, only ever via menu selection. It requires user to input data.
So shouldn't I rather remove the checks for INTERACTIVE ? I just kept them for safety reasons.

Reply:
Then just write out that it will be ignored while called UNATTENDED.

Reply:
ok